components/Cta/Cta.styles.ts

```diff
@@ -74,11 +80,11 @@ export const ctaSizes: CtaSizeObjectType = {
   default: 'pt-9 pb-10 pl-18 pr-16 lg:pl-22 lg:pr-20 lg:pt-10 lg:pb-11 text-16 lg:text-20',
   large: 'pl-28 pr-26 pt-16 pb-17 lg:pr-40 lg:pl-44 lg:pt-20 lg:pb-22 text-18 lg:text-24',
   link: 'text-16 lg:text-20',
+  mainNavLink: 'text-19 md:text-20 2xl:text-24',
   mastheadGiving: 'text-14 sm:text-15 md:text-16 lg:text-19',
-  back: 'text-16',
   close: 'text-18 md:text-21',
   chip: 'py-7 px-22 text-18',
-  storyCardTag: 'text-16 lg:text-18',
+  storyCardChip: 'text-15 py-6 px-16 lg:text-17 lg:pt-8 lg:pb-7 lg:px-18',
```

Comment:
These are from story-list-page integration branch that I didn't take out. They're just new styles for the future and won't affect anything on production.